src/App.tsx

```diff
@@ -28,21 +29,34 @@ const PullRequestPage = lazy(() => import('./pages/PullRequestPage'))
 const RepoPage = lazy(() => import('./pages/RepoPage'))
 const SyncProviderPage = lazy(() => import('./pages/SyncProviderPage'))
 
+interface URLParams {
+  provider: string
+}
+
 const HomePageRedirect = () => {
-  const { provider } = useParams()
+  const { provider } = useParams<URLParams>()
   const { data: currentUser } = useUser()
+  const { data: internalUser } = useInternalUser({})
   const { params } = useLocationParams()
 
-  if (!provider || !currentUser) {
-    return <Redirect to="/login" />
+  let redirectURL = '/login'
+
+  if (internalUser) {
+    redirectURL = `/${internalUser.owners![0]?.service}/${
+      internalUser.owners![0]?.username
```

Comment:
Same suggestion here
```suggestion
      internalUser.owners?.[0]?.username
```

Reply:
You could also toss a lodash `isArray` in the if condition, if that's more your style

Reply:
Yeah, my IDE didn't like that one unfortunately 😭 

What do you think about just changing the if statement to 

`  if (internalUser && internalUser.owners) {
    redirectURL = `/${internalUser.owners[0]?.service}/${internalUser.owners[0]?.username}`
  }`

Reply:
Think its a _litttttttle_ cleaner that way?

Reply:
oh yea that's great too!